### DISCUSSION TITLE
Podman dependency on Homebrew

### DISCUSSION THREAD

**Discussion by ajram23** [containers/podman#23652] [General]
I am trying to package a container within my swift app and I am trying to use podman. However podman requires homebrew install which then requires command line tools for Xcode. Is there a light weight option to ship a container without needing homebrew or command line tools for Xcode. anyone has been successful in doing this with or without podman? Any help is greatly appreciated!

**Comment by rhatdan** [comment]
Podman installed from the github.com repository should not require homebrew.

**Reply by ajram23** [reply]
Could you please elaborate? Any docs I might be to look at ?

**Reply by rhatdan** [reply]
https://podman.io/docs/installation
